app/controllers/staff/activity_financials_controller.rb

```diff
@@ -16,6 +16,6 @@ def show
     @budget_presenters = @budgets.includes(:parent_activity, :providing_organisation).map { |budget| BudgetPresenter.new(budget) }
     @planned_disbursement_presenters = @planned_disbursements.map { |planned_disbursement| PlannedDisbursementPresenter.new(planned_disbursement) }
     @implementing_organisation_presenters = activity.implementing_organisations.map { |implementing_organisation| ImplementingOrganisationPresenter.new(implementing_organisation) }
-    @transfers = policy_scope(activity.source_transfers).map { |transfer| TransferPresenter.new(transfer) }
+    @transfers = policy_scope(activity.source_transfers).map { |transfer| OutgoingTransferPresenter.new(transfer) }
```

Comment:
This is up to you, but would it help or hinder to rename the instance variables(s) too?

Reply:
I did consider it, but there's going to be bunch of work coming down the pipe with concerns etc, so I didn't think it was worth doing at the time